lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -21,6 +21,96 @@ class Proxy
      */
     protected static $_profiles = array();
 
+    protected static $services = null;
+
+    protected static $appContext = null;
+
+    public static function setServices($services)
```

Comment:
Type of parameters is missing (in an `@param` or directly into the code).
Please document parameters and return values for all new methods.

Reply:
the syntax is `@param <type> <parameter name>`

`@param \lizmapServices $services`. No two `@param` for the same parameter.